dist/checkout-button.js

```diff
@@ -1,2 +1,2 @@
<s;c++){var u=a[c];if(!t.hasOwnProperty(u))return!1;if(!r(e[u],t[u],n))return!1}return!0}(e,t,n))):e===t)}n.d(t,"a",(function(){return r}))},function(e,t){e.exports=require("@bigcommerce/form-poster")},function(e,t,n){"use strict";n.d(t,"a",(function(){return i}));var r=n(56);function i(e,t){var n=e.firstName||t.firstName||"",i=e.lastName||t.lastName||"";return{addresses:(e.addresses||[]).map((function(e){return Object(r.a)(e)})),customerId:e.id,isGuest:e.isGuest,storeCredit:e.storeCredit,email:e.email||t.email||"",firstName:n,lastName:i,name:e.fullName||[n,i].join(" "),customerGroupName:e.customerGroup&&e.customerGroup.name}}},function(e,t,n){"use strict";n.d(t,"a",(function(){return s}));var r=n(0),i=n(1),a=n(62),o=n(73),c=n(108);n(65);function s(e,t){void 0===t&&(t={});var n,r,s=e.currency.decimalPlaces,u=new o.a(s);return{id:e.orderId,items:Object(a.a)(e.lineItems,e.currency.decimalPlaces,"productId"),orderId:e.orderId,currency:e.currency.code,customerCanBeCreated:e.customerCanBeCreated,payment:p(e.payments,t.payment),subtotal:{amount:e.baseAmount,integerAmount:u.toInteger(e.baseAmount)},coupon:{discountedAmount:Object(i.reduce)(e.coupons,(function(e,t){return e+t.discountedAmount}),0),coupons:e.coupons.map(c.a)},discount:{amount:e.discountAmount,integerAmount:u.toInteger(e.discountAmount)},token:t.orderToken,callbackUrl:t.callbackUrl,discountNotifications:[],giftCertificate:(n=e.payments,r=Object(i.filter)(n,{providerId:"giftcertificate"}),{totalDiscountedAmount:Object(i.reduce)(r,(function(e,t){return t.amount+e}),0),appliedGiftCertificates:Object(i.keyBy)(r.map((function(e){return{code:e.detail.code,discountedAmount:e.amount,remainingBalance:e.detail.remaining,giftCertificate:{balance:e.amount+e.detail.remaining,code:e.detail.code,purchaseDate:""}}})),"code")}),socialData:h(e),status:e.status,hasDigitalItems:e.hasDigitalItems,isDownloadable:e.isDownloadable,isComplete:e.isComplete,shipping:{amount:e.shippingCostTotal,integerAmount:u.toInteger(e.shippingCostTotal),amountBeforeDiscount:e.shippingCostBeforeDiscount,integerAmountBeforeDiscount:u.toInteger(e.shippingCostBeforeDiscount)},storeCredit:{amount:d(e.payments)},taxes:e.taxes,taxTotal:{amount:e.taxTotal,integerAmount:u.toInteger(e.taxTotal)},handling:{amount:e.handlingCostTotal,integerAmount:u.toInteger(e.handlingCostTotal)},grandTotal:{amount:e.orderAmount,integerAmount:e.orderAmountAsInteger}}}function u(e){return"PAYMENT_STATUS_"+e}function d(e){var t=Object(i.find)(e,{providerId:"storecredit"});return t?t.amount:0}function p(e,t){void 0===t&&(t={});var n=Object(i.find)(e,l);return n?{id:n.providerId,status:u(n.detail.step),helpText:n.detail.instructions,returnUrl:t.returnUrl}:{}}function l(e){return"giftcertificate"!==e.providerId&&"storecredit"!==e.providerId}function h(e){var t={};return Object(r.__spreadArrays)(e.lineItems.physicalItems,e.lineItems.digitalItems).forEach((function(e){var n;t[e.id]=(n=e,["fb","tw","gp"].reduce((function(e,t){var r=n.socialMedia&&Object(i.find)(n.s
```

Comment:
It's `dist` folder, bro🧐

Reply:
🤦

Reply:
My favourite bit of this is that the entire line gets included in the GH notification email, so I had to scroll about 50 pages to see what we were even discussing.

Reply:
Just had a look at the email the `This message seems dangerous`, definitely helps in making sure the user scrolls to the end of the email.